packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -429,6 +429,15 @@ export default class NewVm extends BaseComponent {
     const resourceSet = this._getResourceSet()
     const { template } = this.props
 
+    // Either use `memory` OR `memory*` params
+    let { memory, memoryStaticMax, memoryDynamicMin, memoryDynamicMax } = state
+    if ((state.memoryStaticMax != null || state.memoryDynamicMin != null) && memoryDynamicMax == null) {
```

Comment:
Why use `state.memoryStaticMax` and `state.memoryDynamicMin` instead of `memoryStaticMax` and `memoryDynamicMin` ?